README.md

```diff
@@ -162,6 +162,13 @@ server-tools: # Here we repeat server-tools, but no problem because it's a
     - html_text
 ```
 
+You can add all modules in a repo by using a `*`:
+
+```yaml
+website:
+    - "*"
```

Comment:
Another option I'm thinking of right now (sorry for annoying): fully support globs. This way you can:

```yaml
event:
  - event_*
website:
  - *
```

I think I like this one most!